frontend-student/src/components/home/Home.jsx

```
@@ -313,12 +311,11 @@ export default function Home() {
             transition={{ duration: 0.5, delay: 0.4 }}
             viewport={{ once: true }}
           >
-            <button className='px-8 py-3 bg-white text-blue-600 font-bold rounded-lg hover:bg-gray-100 transition'>
-              Sign Up Now
-            </button>
-            <button className='px-8 py-3 bg-transparent border border-white text-white font-bold rounded-lg hover:bg-white/10 transition'>
-              Request Demo
-            </button>
+            <Link to='/signup'>
+              <button className='px-8 py-3 bg-white text-blue-600 font-bold rounded-lg hover:bg-gray-100 transition'>
+                Sign Up Now
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)